### ISSUE TITLE
Unexpected behavior on select, map and proc

### ISSUE THREAD

**Issue by jeromew2** [rubocop/rubocop#13070]
Rubocop format select, map, proc with (&attribute) if there is only one parameter in the condition.

ex: after_save  :something, :if => proc(&:saved_change_to_name?)

This make a bug because rails expect a database attribute.

--------

## Expected behavior

Keep after_save  :something, :if => proc {|b| b.saved_change_to_name?}

## Actual behavior

Transform in after_save  :something, :if => proc(&:saved_change_to_name?)

**Comment by koic** [comment]
RuboCop is a static analysis tool for the Ruby Programming Language. The issue is caused by behavior that is extended by Active Support. You can switch to behavior that corresponds to the extensions provided by Active Support by using rubocop-rails or setting `AllCops:ActiveSupportExtensionsEnabled: true`:

```yaml
# .rubocop.yml
AllCops:
  ActiveSupportExtensionsEnabled: true
```

https://docs.rubocop.org/rubocop/1.65/cops_style.html#allcopsactivesupportextensionsenabled-false-default-3
